Bottom: 64bb76c87124025e2a1a803435e23f18f0cc84d5
Top:    29fdbfc2381ca6238bb2fcee970c7ec5725542a9
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:25:33 +0800

Refresh of autofs-5.0.7-fix-portmap-version.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index f8b5d19..aea47d3 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -78,7 +78,7 @@
 - fix bad mkdir permission on create.
 - setup program map env from macro table.
 - add short host name standard marco variable.
-- fix portmap version.
+- fix portmap lookup.
 
 25/07/2012 autofs-5.0.7
 =======================
